elastic-search-support/src/main/resources/globalSearchCasesMapping.json

```diff
@@ -0,0 +1,195 @@
+{
+  "properties": {
+    "caseReference": {
+      "type": "text",
+      "fields": {
+        "keyword": {
+          "type": "keyword",
+          "ignore_above": 256,
+          "normalizer": "lowercase_normalizer"
+        }
+      }
+    },
+    "jurisdictionId": {
+      "type": "text",
+      "fields": {
+        "keyword": {
+          "type": "keyword",
+          "ignore_above": 256,
+          "normalizer": "lowercase_normalizer"
+        }
+      }
+    },
+    "caseTypeId": {
+      "type": "text",
+      "fields": {
+        "keyword": {
+          "type": "keyword",
+          "ignore_above": 256,
+          "normalizer": "lowercase_normalizer"
+        }
+      },
+      "analyzer": "standard",
+      "search_analyzer": "alphaNumericStringAnalyzer"
```

Comment:
Looks like we're missing this for some of the other text fields?

Reply:
done